.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,42 +38,61 @@ body:
   attributes:
     label: System information
     description: |
-      - Specify the OS version, and when relevant hardware information.
-      - For issues that are likely OS-specific and/or graphics-related, please specify the CPU model and architecture.
-      - For graphics-related issues, specify the GPU model, driver version, and the rendering backend (GLES2, GLES3, Vulkan).
-      - **Bug reports not including the required information may be closed at the maintainers' discretion.** If in doubt, always include all the requested information; it's better to include too much information than not enough information.
-      - **Starting from Godot 4.1, you can copy this information to your clipboard by using *Help > Copy System Info* at the top of the editor window.**
-    placeholder: Windows 10 - Godot v4.0.3.stable - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 970 (nvidia, 510.85.02) - Intel Core i7-10700KF CPU @ 3.80GHz (16 Threads)
+      - Copy and paste system information by clicking on
+        *Help > Copy System Info* at the top of the editor window.
+      - For Godot 4.0 and prior, see [sharing system information](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#sharing-system-information).
+    placeholder: |
+      Godot v4.2.1.stable - Windows 11 - Vulkan (Forward+) - dedicated AMD
+      Radeon RX Vega (RADV VEGA10) () - AMD Ryzen 7 5800X 8-Core Processor
+      (16 Threads)
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Issue description
     description: |
-      Describe your issue briefly. What doesn't work, and how do you expect it to work instead?
-      You can include images or videos with drag and drop, and format code blocks or logs with <code>\`\`\`</code> tags, on separate lines before and after the text. (Use <code>\`\`\`gdscript</code> to add GDScript syntax highlighting.)
-      Please do not add code examples or error messages as screenshots, but as text, this helps searching for issues and testing the code. If you are reporting a bug in the editor interface, like the script editor, please provide both a screenshot *and* the text of the code to help with testing.
+      Describe your issue briefly.
```

Comment:
Might it be recommended to include the expected behavior as well as the problem you are currently facing? Maybe it should be a separate item as “What you wanted”. That information will clarify whether it is a bug or should be treated as a enhancement/proposal.